Vagrantfile

```diff
@@ -129,10 +129,12 @@ Vagrant.configure("2") do |config|
     end
   end
 
-    config.vm.define "dev" do |nodeconfig|
-      nodeconfig.vm.box = "ubuntu/impish64"
+    config.vm.define "elastic-agent" do |nodeconfig|
+      nodeconfig.vm.box = "ubuntu/jammy64"
 
-      nodeconfig.vm.hostname = "elastic-agent-dev"
+      # We deliberately set a fully-qualified domain name for the VM; it helps
+      # test the FQDN feature flag.
+      nodeconfig.vm.hostname = "elastic-agent-dev.elastic.dev.internal"
```

Comment:
Odd that these lines are showing up in this diff as they seem to already exist on `main`: https://github.com/elastic/elastic-agent/blob/dfd9554406f581660747663d15476c636095fa20/Vagrantfile#L132-L137